internal/pkg/service/stream/storage/node/coordinator/coordinator.go

```diff
@@ -17,6 +17,11 @@ import (
 func Start(ctx context.Context, d dependencies.CoordinatorScope, cfg config.Config) error {
 	logger := d.Logger().WithComponent("storage.node.coordinator")
 	logger.Info(ctx, `starting storage coordinator node`)
+	// Change default bridge behaviour to use Mirror as it is required for sink throttling
+	err := d.KeboolaSinkBridge().MirrorJobs(d)
```

Comment:
Shouldn't you pass `ctx` here and use `withoutCancel` inside `MirrorJobs` instead of `context.Background()`?

Reply:
6cd186a0915f6f6fb59febd0e09a3815958f9018